edx_exams/apps/lti/views.py

```diff
@@ -172,7 +269,8 @@ def start_proctoring(request, attempt_id):
         attempt_number=attempt.attempt_number,
         start_assessment_url=proctoring_start_assessment_url,
         assessment_control_url=assessment_control_url,
-        assessment_control_actions=['flagRequest'],
+        # TODO: Is this the correct action?
+        assessment_control_actions=['terminateRequest'],
```

Comment:
it's actually just `terminate` the LTI docs are missing a space making it seems like 'Request' is part of the string.